neo/test/iotest/test_alphaomegaio.py

```diff
@@ -13,8 +13,7 @@ class TestAlphaOmegaIO(BaseTestIO, unittest.TestCase):
         'alphaomega'
     ]
     entities_to_test = [
-        'alphaomega/File_AlphaOmega_1.map',
```

Comment:
So the `map` files are not supported any more? I think this would be worth a sentence in the docstring, i.e 'For loading of .map files use an old Neo version (<0.11.0)' @samuelgarcia Do you agree?